container/check_container.go

```diff
@@ -143,6 +143,14 @@ func WithInsecureConnection() Option {
 	}
 }
 
+// WithManifestListDigest signifies that we have a manifest list and should add
+// this digest to any Pyxis calls
+func WithManifestListDigest(manifestListDigest string) Option {
+	return func(cc *containerCheck) {
+		cc.manifestListDigest = manifestListDigest
+	}
+}
+
```

Comment:
1) This comment is a bit unclear as to how this changes container certification for a library consumer. Specifically because library consumers don't have submission hooks.
2) Should we validate that this value is a valid digest, or is that done elsewhere?

Reply:
1. I'll add some more verbiage. This is an unfortunate side-effect. I will explore if this can be made available only to preflight itself, and not confuse library users.
2. The assumption here is that it's validated elsewhere. Preflight itself is not using this directly as a digest.